src/Portal/docs/Set-AzPortalDashboard.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Can we add a new parameter set in Update-AzPortalDashboard instead of using a Set-* cmdlet

Reply:
This was discussed in the design review - since using the template file replaces the entire representation of the Dashboard, only the 'Set' verb makes sense.